datasets/ONS-International-exports-of-services-from-subnational-areas-of-the-UK/main.py

```diff
@@ -17,106 +17,279 @@
 
 # +
 from gssutils import *
+import json
 
-scraper = Scraper('https://www.ons.gov.uk/businessindustryandtrade/internationaltrade/datasets/internationalexportsofservicesfromsubnationalareasoftheuk')
-tabs = { tab.name: tab for tab in scraper.distributions[0].as_databaker() }
+cubes = Cubes("info.json")
+trace = TransformTrace()
+info = json.load(open('info.json'))
+scraper = Scraper(seed = 'info.json')
+scraper
 # -
 
+tabs = { tab.name: tab for tab in scraper.distributions[0].as_databaker() }
 list(tabs)
 
-next_table = pd.DataFrame()
+data_download = scraper.distributions[0].downloadURL
+datasetTitle = "International exports of services from subnational areas of the UK"
```

Comment:
use `distribution.title` to retrieve title, helps to ensure no human error. 
e.g. :
`datasetTitle = distribution.title`

Reply:
Hardcoding of title is replaced by code, which ensure's correct title is used.